src/components/Manual_Override.tsx

```diff
@@ -1,22 +1,106 @@
-import { Button, Flex, Text } from "@chakra-ui/react";
+import {
+  Button,
+  Flex,
+  Text,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  Input,
+  useToast,
+} from "@chakra-ui/react";
+import axiosInstance from "../axiosInstance";
+import { SetStateAction, useState } from "react";
 
 interface OverrideProps {
   changePage: (pageNumber: number) => void;
 }
 
 export default function OverridePage({ changePage }: OverrideProps) {
+  const toast = useToast();
+  const [input, setInput] = useState("");
+  const [isError, setIsError] = useState(false);
+  const handleInputChange = (e: {
+    target: { value: SetStateAction<string> };
+  }) => {
+    const value = e.target.value;
+    setInput(value);
+    if (value) {
+      setIsError(false);
+    }
+  };
+  const handleManualOverride = async () => {
+    if (input != "") {
+      try {
+        const response = await axiosInstance.post("/api/admin/walkin", {
+          email: input,
+        });
+        setInput("");
+        toast({
+          title: "Email link successfully sent",
+          status: "success",
+          isClosable: true,
+        });
+        console.log(response.data);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      } catch (e: any) {
+        toast({
+          title: e.response.data.fallbackMessage || "An unknown error occured",
+          status: "error",
+          isClosable: true,
+        });
+      }
```

Comment:
I think having the toast appear at the top would be better to not cover up the button on the bottom.

https://v2.chakra-ui.com/docs/components/toast/usage#changing-the-toast-position